.gitpod.Dockerfile

```diff
@@ -0,0 +1,19 @@
+#Pull Gitpod Workspace Image
+FROM gitpod/workspace-full
+
+#Pull Postgres Image
+FROM gitpod/workspace-postgres
+
+USER gitpod
+
+# Set up Java environment
+RUN bash -c '. /home/gitpod/.sdkman/bin/sdkman-init.sh && \
+    sdk install java 17.0.3-ms && \
+    sdk default java 17.0.3-ms'
+
+# Set up Node.js environment
+RUN bash -c 'VERSION="18.14.0" \
+    && source $HOME/.nvm/nvm.sh && nvm install $VERSION \
+    && nvm use $VERSION && nvm alias default $VERSION'
```

Comment:
Shouldn't we do the same for Maven to make sure of the version it is using?

Reply:
Thank you @alecharp, I have resolved this by pushing another commit which will ensure the maven version to be 3.9.0